src/sentry/api/endpoints/project_platforms.py

```diff
@@ -2,17 +2,17 @@
 
 from rest_framework.response import Response
 from sentry.api.bases.project import ProjectEndpoint
-from sentry.api.serializers import serialize, register, Serializer
 from sentry.models import ProjectPlatform
-
-
-@register(ProjectPlatform)
-class ProjectPlatformSerializer(Serializer):
-    def serialize(self, obj, attrs, user):
-        return {'platform': obj.platform, 'dateCreated': obj.date_added}
+from rest_framework import status
+from sentry.api.serializers import serialize
 
 
 class ProjectPlatformsEndpoint(ProjectEndpoint):
     def get(self, request, project):
-        queryset = ProjectPlatform.objects.filter(project_id=project.id)
-        return Response(serialize(list(queryset), request.user))
+
+        project_platforms = ProjectPlatform.objects.filter(project_id=project.id)
+
+        if len(project_platforms) < 1:
+            return Response({}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Should this be a 400? Not sure if "no platform" means bad request, so much as it's just an empty 200?

Reply:
This `len(project_platforms)` is also going to cause the `QuerySet` to be evaluated twice. You should coerce the `QuerySet` to a list first to explicitly resolve the query once.

```python
project_platforms = list(ProjectPlatform.objects.filter(project_id=project.id))
```

And yeah, I'd agree as well that an empty list should just be a valid response of no items. So you could likely remove this whole block and just let the serializer handle it.